source/fe/fe_wedge_p.cc

```diff
@@ -105,19 +105,37 @@ FE_WedgePoly<dim, spacedim>::FE_WedgePoly(
 {
   AssertDimension(dim, 3);
 
-  if (degree == 1)
-    {
-      for (const unsigned int i : ReferenceCells::Wedge.vertex_indices())
-        this->unit_support_points.emplace_back(
-          ReferenceCells::Wedge.vertex<dim>(i));
-    }
-  else
+  Assert(1 <= degree && degree <= 2, ExcNotImplemented());
+
+  FE_SimplexP<2> fe_triangle(degree);
+  FE_Q<2>        fe_quad(degree);
+
+  for (unsigned int layer = 0; layer <= degree; ++layer)
     {
-      // TODO: Wedge elements work for higher degrees, but we don't currently
-      // fill their support points. Leaving the array empty is valid, however,
-      // and will simply result in an error when someone tries to access the
-      // array.
+      const double z = static_cast<double>(layer) / static_cast<double>(degree);
```

Comment:
I note that this is not generalizable to `degree > 2` where we would use the points of `FE_Q<1>`. Furthermore, I wonder whether the code can actually use the lexicographic ordering of points in the z direction, rather than the hierarchical one. I would have assumed that continuity forces the latter one?

Reply:
You are right. I have fixed it!